ui/components/app/metamask-template-renderer/safe-component-list.js

```diff
@@ -86,6 +87,7 @@ export const safeComponentList = {
   Typography,
   SmartTransactionStatusPage,
   UrlIcon,
+  OriginPill,
```

Comment:
You can see the moment we lost our beautiful alphabetical order 😄

Reply:
Reordered!